code/__DEFINES/dcs/signals/signals_mob/signals_mob_main.dm

```diff
@@ -87,8 +87,10 @@
 	// #define SPEECH_SANITIZE 4
 	#define SPEECH_LANGUAGE 5
 	/* #define SPEECH_IGNORE_SPAM 6
-	#define SPEECH_FORCED 7 */
-	#define SPEECH_RANGE 8
+	#define SPEECH_FORCED 7
+	#define SPEECH_FILTERPROOF 8 */
```

Comment:
where the heck did this come from

Reply:
I suspect what happened is that the `say()` procs args kept scaling to have more and more and this never got properly added as a signal. (filterproof was the most recent say arg before my changes)

Reply:
SPEECH_FILTERPROOF doesn't appear anywhere else in this PR, even as a removal

Reply:
The defines are positional arguments for `say()`.  Not all of them are used, but it's a good idea to include them in case someone needs them later.  It would also look very weird to see:

```
////from /mob/living/say(): ()
#define COMSIG_MOB_SAY "mob_say"
	// used to access COMSIG_MOB_SAY argslist
	#define SPEECH_MESSAGE 1
	#define SPEECH_SPANS 3
	#define SPEECH_LANGUAGE 5
	#define SPEECH_RANGE 9
	#define SPEECH_SAYMODE 10
```
vs
```
////from /mob/living/say(): ()
#define COMSIG_MOB_SAY "mob_say"
	// used to access COMSIG_MOB_SAY argslist
	#define SPEECH_MESSAGE 1
	// #define SPEECH_BUBBLE_TYPE 2
	#define SPEECH_SPANS 3
	// #define SPEECH_SANITIZE 4
	#define SPEECH_LANGUAGE 5
	/* #define SPEECH_IGNORE_SPAM 6
	#define SPEECH_FORCED 7
	#define SPEECH_FILTERPROOF 8 */
	#define SPEECH_RANGE 9
	#define SPEECH_SAYMODE 10
```

Having gaps in the arguments is confusing.  It's better to just keep them there commented out.

Reply:
can we just remove commented out defines?

Reply:
Gutted with 0b65f114cf2667a131a8121fb47624b37a070ad1